src/actions/articleActions.js

```diff
@@ -113,3 +115,28 @@ export const likeDislike = (payload, slug) => dispatch => {
       return toast.error(error.response.data.detail, { autoClose: false, hideProgressBar: true });
     });
 };
+
+export const updateArticle = (slug, newData) => dispatch => {
+  toast.dismiss();
+  dispatch(editArticleInititated(true));
+  axiosInstance
+    .put(`/api/articles/${slug}/`, newData)
+    .then(response => {
+      dispatch(editArticleSuccess(true));
+      toast.success(
+        response.data.message,
+        { autoClose: 3500, hideProgressBar: true },
+      );
+    })
+    .catch((error) => {
+      let errorMessage = '';
+      if (error.response.status === 403) {
+        errorMessage = 'Re-login and try again';
+      }
+      if (error.response.status === 404) {
+        errorMessage = 'Please enter valid text';
+      }
+      dispatch(editArticleError(errorMessage));
+      toast.error(errorMessage, { autoClose: false, hideProgressBar: true });
```

Comment:
If you are notifying the user of this error, why add it to state?

Reply:
The dispatch of the error message to the state has been disabled. The user will receive the error as a toast.